### PR TITLE
chore(ghactions): add workflow to automatically assign reviewers

### PR DESCRIPTION
**Pull request by platten**

### CODE REVIEW

**Comment by haraldh** [comment]
your git message still has a capital `Add`

**Comment by platten** [comment]
Yes and no. The goal is not to add all 3 CODEOWNERS to each review, but to
add 1 of the 3 :)

File: .github/reviewer-lottery.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+groups:
+  - name: codeowners
+    reviewers: 1
+    internal_reviewers: 1
+    usernames:
+      - npmccallum
+      - haraldh
+      - rvolosatovs
```
[the remark this answers is not in the document]
Comment (platten) [reply .github/reviewer-lottery.yml:8]: Yes and no. The goal is not to add all 3 CODEOWNERS to each review, but to
add 1 of the 3 :)

[the remark this answers is not in the document]
Comment (haraldh) [reply .github/reviewer-lottery.yml:8]: hmm, I cannot even unassign CODEOWNERS from the Reviewers, so I wonder if that works

[the remark this answers is not in the document]
Comment (platten) [reply .github/reviewer-lottery.yml:8]: I'll just need to adjust the appropriate branch protection rule and it should be good.

File: .github/workflows/assign-reviewers.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    types: [opened, ready_for_review, reopened]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: uesteibar/reviewer-lottery@v2
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (rvolosatovs) [inline .github/workflows/assign-reviewers.yml:13]: Should we limit the permissions of this token https://docs.github.com/en/actions/security-guides/automatic-token-authentication#modifying-the-permissions-for-the-github_token ?

**Comment by platten** [comment]
Closing this as it does not meet the requirements of a multi-phase review and approval workflow.

**Pull request closed without merging**